README.md

```diff
@@ -8,15 +8,81 @@
 
 ## Overview
 
-[//]: # (TODO: Add overview mentioning the purpose of the module, supported REST API versions, and other high-level details.)
+[Dropbox](https://www.dropbox.com/) cloud storage service that allows users to store, share, and manage files online.
+
+The `ballerinax/dropbox` package offers APIs to connect and interact with [Dropbox API](https://www.dropbox.com/developers/documentation/http/documentation) endpoints, specifically based on [Dropbox API v2](https://dropbox.github.io/dropbox-api-v2-explorer/).
+
 
 ## Setup guide
 
-[//]: # (TODO: Add detailed steps to obtain credentials and configure the module.)
+To use the Dropbox connector, you must have access to the Dropbox API. Dropbox uses OAuth 2.0, an open specification, to authorize access to data. To get an OAuth token from Dropbox to enable Dropbox Connector to access your Dropbox account via the API you’ll need to create a new app on the DBX Platform.
+
+### Creating an App on the DBX Platform
+
+Navigate to https://www.dropbox.com/developers/apps and select “Create app”. 
```

Comment:
Shall we elaborate on the steps more, by giving UI examples?
you can find the reference here : https://github.com/ballerina-platform/module-ballerinax-twitter/blob/main/README.md